src/processor_x86.cpp

```diff
@@ -286,6 +288,7 @@ static constexpr CPUSpec<CPU, feature_sz> cpus[] = {
 
     {"znver1", CPU::amd_znver1, CPU::generic, 0, Feature::znver1},
     {"znver2", CPU::amd_znver2, CPU::generic, 0, Feature::znver2},
+    {"znver3", CPU::amd_znver3, CPU::generic, 0, Feature::znver3},
```

Comment:
`{"znver3", CPU::amd_znver3, CPU::amd_znver2, 130000, Feature::znver3}`

Note that this is one of the main reasons the code is repeated. I don't think any other project carries a database for LLVM versions.

Reply:
Aha. Admittedly, i did not understood this particular line indeed.
Let me see if that helps.

Reply:
These are the fallback to support older llvm versions.